setupbase.py

```diff
@@ -388,6 +390,7 @@ def _record_commit(self, base_dir):
             repo_commit = repo_commit.strip().decode("ascii")
 
             out_pth = pjoin(base_dir, pkg_dir, 'utils', '_sysinfo.py')
+            out_pth = Path(out_pth)
             if os.path.isfile(out_pth) and not repo_commit:
```

Comment:
Same here, we want to avoid pjoin, 


```suggestion
            out_pth = Path(base_dir)/  pkg_dir/ 'utils'/ '_sysinfo.py'
            if os.path.isfile(out_pth) and not repo_commit:
```

And then you can likely go further, is `out_path` is a `Path`, you can the use `out_path.is_file()`

Reply:
Thank you for the kind words. This was my first contribution and the experience so far is good :)
I didn't know we could use a `/` for joining paths. Guess we learn something new everyday. I will commit the requested changes right away.